test/SILOptimizer/static_bigint_enums.swift

```diff
@@ -0,0 +1,28 @@
+// RUN: %target-build-swift -parse-as-library -O %s -module-name=test -emit-sil | %FileCheck %s
+
+// RUN: %empty-directory(%t) 
+// RUN: %target-build-swift -parse-as-library -O -module-name=test %s -o %t/a.out
+// RUN: %target-run %t/a.out | %FileCheck %s -check-prefix=CHECK-OUTPUT
+
+// REQUIRES: executable_test,swift_stdlib_no_asserts,optimized_stdlib
+// UNSUPPORTED: use_os_stdlib
+
+
+@available(SwiftStdlib 5.8, *)
+public enum StaticIntStringEnum {
+  case s(StaticString)
+  case i(StaticBigInt)
+
+  // CHECK-LABEL: sil_global hidden @$s4test19StaticIntStringEnumO1xACvpZ : $StaticIntStringEnum = {
```

Comment:
If you pass `-check-prefix` to FileCheck, I believe it suppresses the default `CHECK` prefix.

Reply:
yes. This check line is for the first compiler run (see line 1)